PEP8 compliance

I think it should be included in setup.py some linting tool

pathcl@ladrillo:~/src/vsphere-python-restapi$ flake8 --statistics .
./myapi.py:1:1: F401 'url_for' imported but unused
./myapi.py:3:1: F401 'json' imported but unused
./myapi.py:10:1: E265 block comment should start with '# '
./myapi.py:10:40: W291 trailing whitespace
./myapi.py:13:1: E265 block comment should start with '# '
./myapi.py:14:1: E265 block comment should start with '# '
./myapi.py:15:1: E265 block comment should start with '# '
./myapi.py:17:1: E302 expected 2 blank lines, found 1
./myapi.py:21:1: E302 expected 2 blank lines, found 1
./myapi.py:25:1: E302 expected 2 blank lines, found 1
./myapi.py:29:1: E101 indentation contains mixed spaces and tabs
./myapi.py:29:1: W191 indentation contains tabs
./myapi.py:29:19: E251 unexpected spaces around keyword / parameter equals
./myapi.py:29:21: E251 unexpected spaces around keyword / parameter equals
./myapi.py:30:1: E101 indentation contains mixed spaces and tabs
./myapi.py:31:26: E251 unexpected spaces around keyword / parameter equals
./myapi.py:31:28: E251 unexpected spaces around keyword / parameter equals
./myapi.py:33:1: E302 expected 2 blank lines, found 1
./myapi.py:39:44: E231 missing whitespace after ','
./myapi.py:45:1: E303 too many blank lines (3)
./methods.py:18:5: E225 missing whitespace around operator
./methods.py:19:5: E225 missing whitespace around operator
./methods.py:20:4: E225 missing whitespace around operator
./methods.py:30:1: E302 expected 2 blank lines, found 1
./methods.py:33:34: W291 trailing whitespace
./methods.py:34:1: E302 expected 2 blank lines, found 1
./methods.py:40:44: E231 missing whitespace after ','
./methods.py:40:54: E231 missing whitespace after ','
./methods.py:42:21: F841 local variable 'ex' is assigned to but never used
./methods.py:48:14: W291 trailing whitespace
./methods.py:52:1: E302 expected 2 blank lines, found 1
./methods.py:52:43: E231 missing whitespace after ','
./methods.py:58:5: F841 local variable 'vm_json' is assigned to but never used
./methods.py:66:39: E231 missing whitespace after ','
./methods.py:73:39: E231 missing whitespace after ','
./methods.py:83:1: E302 expected 2 blank lines, found 1
./methods.py:87:1: E101 indentation contains mixed spaces and tabs
./methods.py:87:1: W191 indentation contains tabs
./methods.py:88:1: E101 indentation contains mixed spaces and tabs
./methods.py:88:21: E225 missing whitespace around operator
./methods.py:90:1: E101 indentation contains mixed spaces and tabs
./methods.py:90:1: W191 indentation contains tabs
./methods.py:90:2: E265 block comment should start with '# '
./methods.py:91:1: W191 indentation contains tabs
./methods.py:92:1: W191 indentation contains tabs
./methods.py:92:2: E101 indentation contains mixed spaces and tabs
./methods.py:93:1: E101 indentation contains mixed spaces and tabs
./methods.py:95:1: E101 indentation contains mixed spaces and tabs
./methods.py:95:1: W191 indentation contains tabs
./methods.py:96:1: W191 indentation contains tabs
./methods.py:97:1: W191 indentation contains tabs
./methods.py:97:3: E101 indentation contains mixed spaces and tabs
./methods.py:97:23: E231 missing whitespace after ','
./methods.py:97:25: E231 missing whitespace after ','
./methods.py:99:1: E101 indentation contains mixed spaces and tabs
./methods.py:108:1: E302 expected 2 blank lines, found 1
./methods.py:115:26: E251 unexpected spaces around keyword / parameter equals
./methods.py:115:28: E251 unexpected spaces around keyword / parameter equals
./methods.py:116:28: E251 unexpected spaces around keyword / parameter equals
./methods.py:116:30: E251 unexpected spaces around keyword / parameter equals
./methods.py:117:37: E231 missing whitespace after ':'
./methods.py:118:34: E231 missing whitespace after ':'
./methods.py:119:32: E231 missing whitespace after ':'
./methods.py:123:31: E251 unexpected spaces around keyword / parameter equals
./methods.py:123:33: E251 unexpected spaces around keyword / parameter equals
./methods.py:125:25: E251 unexpected spaces around keyword / parameter equals
./methods.py:125:27: E251 unexpected spaces around keyword / parameter equals
./methods.py:130:1: E302 expected 2 blank lines, found 1
./methods.py:135:28: E203 whitespace before ':'
./methods.py:135:38: E231 missing whitespace after ':'
./methods.py:139:1: E302 expected 2 blank lines, found 1
./methods.py:140:5: E265 block comment should start with '# '
./methods.py:149:5: E265 block comment should start with '# '
./methods.py:152:5: E265 block comment should start with '# '
./methods.py:159:1: E101 indentation contains mixed spaces and tabs
./methods.py:159:1: W191 indentation contains tabs
./methods.py:160:1: W191 indentation contains tabs
./methods.py:161:1: E101 indentation contains mixed spaces and tabs
./methods.py:161:1: W293 blank line contains whitespace
./methods.py:162:5: E265 block comment should start with '# '
./methods.py:165:1: W293 blank line contains whitespace
./methods.py:168:80: E501 line too long (90 > 79 characters)
./methods.py:169:1: E302 expected 2 blank lines, found 1
./methods.py:169:25: E231 missing whitespace after ','
./methods.py:170:5: E265 block comment should start with '# '
./methods.py:173:5: E265 block comment should start with '# '
./methods.py:178:1: E101 indentation contains mixed spaces and tabs
./methods.py:178:1: W191 indentation contains tabs
./methods.py:179:1: E101 indentation contains mixed spaces and tabs
./methods.py:179:1: W293 blank line contains whitespace
./methods.py:181:80: E501 line too long (81 > 79 characters)
./methods.py:182:1: E101 indentation contains mixed spaces and tabs
./methods.py:182:1: W191 indentation contains tabs
./methods.py:184:1: E101 indentation contains mixed spaces and tabs
./methods.py:187:1: E302 expected 2 blank lines, found 1
./methods.py:193:5: E265 block comment should start with '# '
./methods.py:195:5: E265 block comment should start with '# '
./methods.py:198:1: E101 indentation contains mixed spaces and tabs
./methods.py:198:1: W191 indentation contains tabs
./methods.py:199:1: W191 indentation contains tabs
./methods.py:199:2: E101 indentation contains mixed spaces and tabs
./methods.py:200:1: W191 indentation contains tabs
./methods.py:201:1: W191 indentation contains tabs
./methods.py:202:1: W191 indentation contains tabs
./methods.py:202:35: E225 missing whitespace around operator
./methods.py:203:1: W191 indentation contains tabs
./methods.py:203:33: E225 missing whitespace around operator
./methods.py:204:1: W191 indentation contains tabs
./methods.py:204:80: E501 line too long (101 > 79 characters)
./methods.py:205:1: W191 indentation contains tabs
./methods.py:206:1: W191 indentation contains tabs
./methods.py:207:1: W191 indentation contains tabs
./methods.py:207:2: E101 indentation contains mixed spaces and tabs
./methods.py:208:1: W191 indentation contains tabs
./methods.py:208:2: E101 indentation contains mixed spaces and tabs
./methods.py:209:1: W191 indentation contains tabs
./methods.py:209:80: E501 line too long (86 > 79 characters)
./methods.py:210:1: E101 indentation contains mixed spaces and tabs
./methods.py:210:80: E501 line too long (94 > 79 characters)
./methods.py:212:1: E101 indentation contains mixed spaces and tabs
./methods.py:212:1: W191 indentation contains tabs
./methods.py:213:1: W191 indentation contains tabs
./methods.py:213:2: E101 indentation contains mixed spaces and tabs
./methods.py:214:1: W191 indentation contains tabs
./methods.py:214:2: E101 indentation contains mixed spaces and tabs
./methods.py:216:1: E101 indentation contains mixed spaces and tabs
./methods.py:228:1: E302 expected 2 blank lines, found 1
./methods.py:230:75: E231 missing whitespace after ','
./methods.py:230:80: E501 line too long (89 > 79 characters)
./methods.py:234:1: E101 indentation contains mixed spaces and tabs
./methods.py:234:1: W191 indentation contains tabs
./methods.py:235:1: W191 indentation contains tabs
./methods.py:235:2: E101 indentation contains mixed spaces and tabs
./methods.py:236:1: E101 indentation contains mixed spaces and tabs
./methods.py:239:1: E302 expected 2 blank lines, found 1
./methods.py:245:80: E501 line too long (92 > 79 characters)
./methods.py:245:93: W291 trailing whitespace
./methods.py:253:1: E101 indentation contains mixed spaces and tabs
./methods.py:253:1: W191 indentation contains tabs
./methods.py:254:1: W191 indentation contains tabs
./methods.py:254:2: E101 indentation contains mixed spaces and tabs
./methods.py:257:1: E302 expected 2 blank lines, found 1
./methods.py:258:1: E101 indentation contains mixed spaces and tabs
./methods.py:273:1: E101 indentation contains mixed spaces and tabs
./methods.py:273:1: W191 indentation contains tabs
./methods.py:274:1: E101 indentation contains mixed spaces and tabs
./methods.py:281:80: E501 line too long (82 > 79 characters)
./methods.py:287:1: E101 indentation contains mixed spaces and tabs
./methods.py:287:1: W191 indentation contains tabs
./methods.py:288:1: W191 indentation contains tabs
./methods.py:288:2: E101 indentation contains mixed spaces and tabs
./methods.py:288:44: E231 missing whitespace after ','
./methods.py:289:1: W191 indentation contains tabs
./methods.py:290:1: E101 indentation contains mixed spaces and tabs
./methods.py:295:1: E302 expected 2 blank lines, found 1
./methods.py:315:49: E231 missing whitespace after ','
./methods.py:315:72: E231 missing whitespace after ','
./methods.py:315:80: E501 line too long (94 > 79 characters)
./methods.py:320:1: E101 indentation contains mixed spaces and tabs
./methods.py:320:1: W191 indentation contains tabs
./methods.py:320:11: E128 continuation line under-indented for visual indent
./methods.py:322:1: E101 indentation contains mixed spaces and tabs
./methods.py:330:1: E101 indentation contains mixed spaces and tabs
./methods.py:330:1: W191 indentation contains tabs
./methods.py:331:1: W191 indentation contains tabs
./methods.py:332:1: E101 indentation contains mixed spaces and tabs
./methods.py:334:1: E101 indentation contains mixed spaces and tabs
./methods.py:334:1: W191 indentation contains tabs
./methods.py:335:1: W191 indentation contains tabs
./methods.py:335:2: E101 indentation contains mixed spaces and tabs
./methods.py:337:1: E101 indentation contains mixed spaces and tabs
./methods.py:338:1: E101 indentation contains mixed spaces and tabs
./methods.py:338:1: W191 indentation contains tabs
./methods.py:340:1: W191 indentation contains tabs
./methods.py:341:1: W191 indentation contains tabs
./methods.py:343:1: W191 indentation contains tabs
./methods.py:343:2: E265 block comment should start with '# '
./methods.py:344:1: W191 indentation contains tabs
./methods.py:345:1: E101 indentation contains mixed spaces and tabs
45      E101 indentation contains mixed spaces and tabs
1       E128 continuation line under-indented for visual indent
1       E203 whitespace before ':'
6       E225 missing whitespace around operator
17      E231 missing whitespace after ','
12      E251 unexpected spaces around keyword / parameter equals
14      E265 block comment should start with '# '
17      E302 expected 2 blank lines, found 1
1       E303 too many blank lines (3)
9       E501 line too long (90 > 79 characters)
2       F401 'url_for' imported but unused
2       F841 local variable 'ex' is assigned to but never used
45      W191 indentation contains tabs
4       W291 trailing whitespace
3       W293 blank line contains whitespace

--- a/methods.py
+++ b/methods.py
@@ -42,15 +42,18 @@ if os.getenv("PORT"):
     default_context = ssl._create_default_https_context
     ssl._create_default_https_context = ssl._create_unverified_context
 
+
 def debugger():
     return os.getenv("VCAP_APP_PORT")
 
 # Create a connection to the vcsa
+
+
 def server_connection():
     SI = None
     # Attempt to connect to the VCSA
     try:
-        SI = connect.SmartConnect(host=host,user=user,pwd=pwd,)
+        SI = connect.SmartConnect(host=host, user=user, pwd=pwd,)
         print("Made the connection")
         atexit.register(connect.Disconnect, SI)
     except IOError:
@@ -63,13 +66,14 @@ def server_connection():
 
 # Helper function to so the actually traversal and printing of the vms.
 # Includes templates
+
+
 def print_vm_info(virtual_machine, depth=1, full_vm_list=None):
     """
     Print information for a particular virtual machine or recurse into a
     folder with depth protection
     """
     maxdepth = 20
-    vm_json = {}
     # if this is a group it will have children. if it does, recurse into them
     # and then return
     if hasattr(virtual_machine, 'childEntity'):
@@ -90,11 +94,13 @@ def print_vm_info(virtual_machine, depth=1, full_vm_list=None):
     summary = virtual_machine.summary
     if hasattr(summary.config, 'product'):
         del vars(summary.config)['product']
-    if summary.config.template == False:
+    if summary.config.template is False:
         full_vm_list.append(vars(summary.config))
     return
 
 # Root function for get a full list of vms
+
+
 def get_all_vm_info():
     try:
         service_instance = server_connection()
@@ -102,7 +108,6 @@ def get_all_vm_info():
             print ("Couldn't get the server instance")
         full_vm_list = []
         content = service_instance.RetrieveContent()
-    #vm_json_return = []
         for child in content.rootFolder.childEntity:
             if hasattr(child, 'vmFolder'):
                 datacenter = child
@@ -120,6 +125,8 @@ def get_all_vm_info():
     return 0
 
 # Helper function to print a short list of vm details
+
+
 def print_short_detail_list(vm):
     vm_summary = vm.summary
     a = vm_summary
@@ -131,20 +138,20 @@ def print_short_detail_list(vm):
     del vars(a.guest)['guestId']
     fullData.update(guest=vars(a.guest))
     fullData.update(storage=vars(a.storage))
-    fullData.update({"overallStatus":a.overallStatus})
-    fullData.update({"powerState":a.runtime.powerState})
-    fullData.update({"bootTime":a.runtime.bootTime})
+    fullData.update({"overallStatus": a.overallStatus})
+    fullData.update({"powerState": a.runtime.powerState})
+    fullData.update({"bootTime": a.runtime.bootTime})
 
     # Grab the tags from vm.config
     tags = {}
     for opts in vm.config.extraConfig:
         if opts.key == "Language":
-	    tags.update({opts.key: opts.value})
-	elif opts.key == "User":
-	    tags.update({opts.key: opts.value})
-	elif opts.key == "Application":
-	    tags.update({opts.key: opts.value})
-    fullData.update({"extraConfig":tags})
+            tags.update({opts.key: opts.value})
+        elif opts.key == "User":
+            tags.update({opts.key: opts.value})
+        elif opts.key == "Application":
+            tags.update({opts.key: opts.value})
+    fullData.update({"extraConfig": tags})
 
     b = vars(a.runtime.host.summary.config.product)
     del vars(a.runtime.host.summary.config)['product']
@@ -155,6 +162,8 @@ def print_short_detail_list(vm):
     return fullData
 
 # Find a specific vm based on the instance UUID
+
+
 def find_vm_by_uuid(UUID):
     si = server_connection()
     search_index = si.content.searchIndex
@@ -163,10 +172,12 @@ def find_vm_by_uuid(UUID):
     if vm is None:
         vm = search_index.FindByUuid(None, uuid, True, False)
         if vm is None:
-            return {"not_found" : {"uuid":uuid}}
+            return {"not_found": {"uuid": uuid}}
     return print_short_detail_list(vm)
 
 # Delete a vm from the server based on the uuid
+
+
 def delete_vm_from_server(uuid):
     # Get Server connection
     SI = server_connection()
@@ -196,8 +207,11 @@ def delete_vm_from_server(uuid):
 
     return "VM is destroyed"
 
-# Change the stats of a vm based on the uuid. Currently can only change the cpu and memory
-def change_vm_stats(uuid,specs):
+# Change the stats of a vm based on the uuid. Currently can only change
+# the cpu and memory
+
+
+def change_vm_stats(uuid, specs):
     # Get server object
     SI = server_connection()
 
@@ -213,36 +227,40 @@ def change_vm_stats(uuid,specs):
         tasks.wait_for_tasks(SI, [task])
 
     if 'mem' in specs:
-        task = VM.ReconfigVM_Task(vim.vm.ConfigSpec(memoryMB=long(specs['mem'])))
+        task = VM.ReconfigVM_Task(
+            vim.vm.ConfigSpec(memoryMB=long(specs['mem'])))
         tasks.wait_for_tasks(SI, [task])
 
     return "I fixed it!"
 
 # Helper function to add a netowrk connection to a vm
+
+
 def add_network(vm, si, content, netName="VM Network"):
     spec = vim.vm.ConfigSpec()
     dev_changes = []
     network_spec = vim.vm.device.VirtualDeviceSpec()
     network_spec.operation = vim.vm.device.VirtualDeviceSpec.Operation.add
     network_spec.device = vim.vm.device.VirtualVmxnet3()
-    #network_spec.device.addressType = "GeneratedAutomatically"
     print("Getting a network...")
-    #Get network type
+    # Get network type
     for net in content.rootFolder.childEntity[0].network:
         if net.name == netName:
             if isinstance(net, vim.dvs.DistributedVirtualPortgroup):
                 # Run portgroup code
-                pg_obj = get_obj(content, [vim.dvs.DistributedVirtualPortgroup], netName)
+                pg_obj = get_obj(
+                    content, [vim.dvs.DistributedVirtualPortgroup], netName)
                 dvs_port_connection = vim.dvs.PortConnection()
-                dvs_port_connection.portgroupKey= pg_obj.key
-                dvs_port_connection.switchUuid= pg_obj.config.distributedVirtualSwitch.uuid
+                dvs_port_connection.portgroupKey = pg_obj.key
+                dvs_port_connection.switchUuid = pg_obj.config.distributedVirtualSwitch.uuid
                 network_spec.device.backing = vim.vm.device.VirtualEthernetCard.DistributedVirtualPortBackingInfo()
                 network_spec.device.backing.port = dvs_port_connection
                 break
             elif isinstance(net, vim.Network):
                 # Run plain network code
                 network_spec.device.backing = vim.vm.device.VirtualEthernetCard.NetworkBackingInfo()
-                network_spec.device.backing.network = get_obj(content, [vim.Network], netName)
+                network_spec.device.backing.network = get_obj(
+                    content, [vim.Network], netName)
                 network_spec.device.backing.deviceName = netName
                 break
         else:
@@ -260,9 +278,12 @@ def add_network(vm, si, content, netName="VM Network"):
     tasks.wait_for_tasks(si, task)
 
 # Helper function to get an object refernce
+
+
 def get_obj(content, vimtype, name):
     obj = None
-    container = content.viewManager.CreateContainerView(content.rootFolder,vimtype, True)
+    container = content.viewManager.CreateContainerView(
+        content.rootFolder, vimtype, True)
 
     for view in container.view:
         if view.name == name:
@@ -271,6 +292,8 @@ def get_obj(content, vimtype, name):
     return obj
 
 # Helper function to create a scsi controller for a vm
+
+
 def create_scsi_controller(vm, si):
     spec = vim.vm.ConfigSpec()
     dev_changes = []
@@ -289,6 +312,8 @@ def create_scsi_controller(vm, si):
             return dev
 
 # Helper function to add a disk to a vm
+
+
 def add_disk(vm, si, disk_size=30):
     spec = vim.vm.ConfigSpec()
     unit_number = 0
@@ -320,13 +345,15 @@ def add_disk(vm, si, disk_size=30):
     disk_spec.device.capacityInKB = new_disk_kb
     if controller is None:
         print "Creating new controller"
-        controller = create_scsi_controller(vm,si)
+        controller = create_scsi_controller(vm, si)
     disk_spec.device.controllerKey = controller.key
     dev_changes.append(disk_spec)
     spec.deviceChange = dev_changes
     vm.ReconfigVM_Task(spec=spec)
 
 # Core create vm function that handles generating all the neccasary part
+
+
 def create_new_vm(specs):
     """Creates a dummy VirtualMachine with 1 vCpu, 128MB of RAM.
     :param name: String Name for the VirtualMachine
@@ -368,34 +395,34 @@ def create_new_vm(specs):
     datastore_path = '[' + datastore + '] ' + vm_name
 
     # bare minimum VM shell creation
-    vmx_file = vim.vm.FileInfo(logDirectory=None,snapshotDirectory=None,suspendDirectory=None,
+    vmx_file = vim.vm.FileInfo(logDirectory=None, snapshotDirectory=None, suspendDirectory=None,
                                vmPathName=datastore_path)
 
     config = vim.vm.ConfigSpec(name=specs['name'], memoryMB=long(specs['mem']),
                                numCPUs=int(specs['cpus']), files=vmx_file,
-                   guestId=specs['guestid'], version=str(specs['vm_version']))
+                               guestId=specs['guestid'], version=str(specs['vm_version']))
 
     # Add custom tags
     config.extraConfig = []
     opt = vim.option.OptionValue()
     options_values = {}
-    #if hasattr(specs, "user"):
+    # if hasattr(specs, "user"):
     if 'user' in specs:
         print("Got user: {0}".format(specs['user']))
-        options_values.update({"User":specs['user']})
-    #if hasattr(specs, "language"):
+        options_values.update({"User": specs['user']})
+    # if hasattr(specs, "language"):
     if 'language' in specs:
         print("Got language: {0}".format(specs['language']))
-        options_values.update({"Language":specs['language']})
+        options_values.update({"Language": specs['language']})
     if 'application' in specs:
         print("Got application: {0}".format(specs['application']))
-        options_values.update({"Application":specs['application']})
+        options_values.update({"Application": specs['application']})
 
     for k, v in options_values.iteritems():
         opt.key = k
-	opt.value = v
-	config.extraConfig.append(opt)
-	opt = vim.option.OptionValue()
+        opt.value = v
+        config.extraConfig.append(opt)
+        opt = vim.option.OptionValue()
 
     # Send off creeation task
     print("Creating VM {0}...".format(vm_name))
@@ -409,7 +436,8 @@ def create_new_vm(specs):
     new_vm = content.searchIndex.FindByDatastorePath(datacenter, path)
     if new_vm is not None:
         # Now that the vm shell is created, add a disk to it
-        # If the user requested a specific size, use that, otherwise use default
+        # If the user requested a specific size, use that, otherwise use
+        # default
         print("Found it...now adding a disk...")
         if hasattr(specs, 'disk_size'):
             add_disk(vm=new_vm, si=SI, disk_size=specs['disk_size'])
@@ -424,63 +452,64 @@ def create_new_vm(specs):
         print("...and powering it on!")
         new_vm.PowerOnVM_Task()
 
-        #Respond with the vm summary
+        # Respond with the vm summary
         return print_short_detail_list(new_vm)
     else:
         return "Could not create vm"
 
 # Function to get a single attribute of a vm
+
+
 def get_vm_attribute(uuid, attr):
     print("Searching for {0} in {1}".format(attr, uuid))
     vmStats = find_vm_by_uuid(uuid)
-    #vmStats = json.load(vm)
     return_value = "null"
     break_var = False
     print("Entering Core attrs")
     for key1, value1 in vmStats.iteritems():
         print("key is {0}".format(key1))
         if key1.lower() == attr.lower():
-	    return_value = value1
-	    break_var = True
+            return_value = value1
+            break_var = True
 
-	if key1 == "extraConfig":
-	    print("Searching in {0}".format(key1))
-	    for key2, value2 in value1.iteritems():
-	        if key2.lower() == attr.lower():
-		    return_value = value2
-		    break_var = True
+        if key1 == "extraConfig":
+            print("Searching in {0}".format(key1))
+            for key2, value2 in value1.iteritems():
+                if key2.lower() == attr.lower():
+                    return_value = value2
+                    break_var = True
 
         if key1 == "guest":
-	    print("Searching in {0}".format(key1))
-	    for key2, value2 in value1.iteritems():
-	        if key2.lower() == attr.lower():
-	            return_value = value2
-		    break_var = True
+            print("Searching in {0}".format(key1))
+            for key2, value2 in value1.iteritems():
+                if key2.lower() == attr.lower():
+                    return_value = value2
+                    break_var = True
 
-	elif key1 == "host":
-	    print("Searching in {0}".format(key1))
-	    for key2, value2 in value1.iteritems():
-	        if key2.lower() == attr.lower():
-	            return_value = value2
-		    break_var = True
+        elif key1 == "host":
+            print("Searching in {0}".format(key1))
+            for key2, value2 in value1.iteritems():
+                if key2.lower() == attr.lower():
+                    return_value = value2
+                    break_var = True
 
-		elif key2 == "product":
-		    print("Searching in {0}".format(key2))
-		    for key3, value3 in value2.iteritems():
-		        if key3.lower() == attr.lower():
-			    return_value = value3
-			    break_var = True
-			    break
+                elif key2 == "product":
+                    print("Searching in {0}".format(key2))
+                    for key3, value3 in value2.iteritems():
+                        if key3.lower() == attr.lower():
+                            return_value = value3
+                            break_var = True
+                            break
                 if break_var:
-		    break
+                    break
 
-	elif key1 == "storage":
-	    print("Searching in {0}".format(key1))
-	    for key2, value2 in value1.iteritems():
-	        if key2.lower() == attr.lower():
-	            return_value = value2
-		    break_var = True
+        elif key1 == "storage":
+            print("Searching in {0}".format(key1))
+            for key2, value2 in value1.iteritems():
+                if key2.lower() == attr.lower():
+                    return_value = value2
+                    break_var = True
         if break_var:
-	    break
+            break
 
     return str(return_value)

--- a/myapi.py
+++ b/myapi.py
@@ -1,34 +1,37 @@
-from flask import Flask, url_for, request, jsonify
+from flask import Flask, request, jsonify
 import methods
-import json
 import os
 app = Flask(__name__)
 app.debug = True
 
 port = int(os.getenv('VCAP_APP_PORT', 8080))
 
-#with open('config.json') as data_file:
+# with open('config.json') as data_file:
 #    data = json.load(data_file)
 #
-#host=data["host"]
-#user=data["username"]
-#pwd=data["password"]
+# host=data["host"]
+# user=data["username"]
+# pwd=data["password"]
+
 
 @app.route('/')
 def index():
     return 'Welcome to the VMware REST API!'
 
+
 @app.route('/debug')
 def debug():
     return methods.debugger()
+
 
 @app.route('/vms/', methods=['GET', 'POST'])
 def get_vms():
     if request.method == 'POST':
         specs = request.get_json()
-        return jsonify(vm = methods.create_new_vm(specs))
+        return jsonify(vm=methods.create_new_vm(specs))
     else:
-        return jsonify(vm = methods.get_all_vm_info())
+        return jsonify(vm=methods.get_all_vm_info())
+
 
 @app.route('/vms/<uuid>/', methods=['GET', 'PUT', 'DELETE'])
 def get_vm(uuid):
@@ -36,14 +39,15 @@ def get_vm(uuid):
         return methods.delete_vm_from_server(uuid)
     elif request.method == 'PUT':
         specs = request.get_json()
-        return methods.change_vm_stats(uuid,specs)
+        return methods.change_vm_stats(uuid, specs)
     else:
         return jsonify(methods.find_vm_by_uuid(uuid))
-          
+
+
 @app.route('/vms/<uuid>/<attr>/', methods=['GET'])
-def get_attr(uuid,attr):
+def get_attr(uuid, attr):
     if request.method == 'GET':
-        return methods.get_vm_attribute(uuid,attr)
+        return methods.get_vm_attribute(uuid, attr)
 
 if __name__ == '__main__':
     app.run(host='0.0.0.0', port=port)

--- a/tools/alarm.py
+++ b/tools/alarm.py
@@ -132,7 +132,7 @@ def print_triggered_alarms(entity=None):
     """
     alarms = entity.triggeredAlarmState
     for alarm in alarms:
-        print("#"*40)
+        print("#" * 40)
         # The alarm key looks like alarm-101.host-95
         print("alarm_moref: {0}".format(alarm.key.split('.')[0]))
         print("alarm status: {0}".format(alarm.overallStatus))

